assignment/scripts/function-practice.js

```diff
@@ -49,39 +63,71 @@ console.log( 'isPositive - should say false', isPositive(-3) );
 
 // 6. Function to return the _last_ item in an array. If the 
 //    array is empty, return `undefined`.
-function getLast( array ) {
-
+function getLast(array) {
+  if (array.length === 0){
+    return 'undefined';
+  }
+    return array[array.length-1];
 }
+console.log(getLast([1, 2, 4, 100, 90, 67]));
+console.log(getLast([]))
+
+
 
 // 7. Function to find a value in an array. Return true if the 
 //    value is found and false otherwise. Use a loop;
 //    DO NOT use Array.includes, Array.indexOf, or Array.find 
-function find( value, array ){
-  
+function find(x, array ){
+  for (y of array){
+    if (y === x){
+      return true;
+    } 
+      return false;
+  }
 }
+let arrays = [1, 2, 3]
+console.log(find(1, arrays));
+console.log(find(1, [0, 2, 4]));
 
 // ----------------------
 // Stretch Goals
 // ----------------------
 // 8. Function to check if a letter is the first letter in a 
 //    string. Return true if it is, and false otherwise
 function isFirstLetter(letter, string) {
-
+   if( letter === string.charAt(0)){;
```

Comment:
Similarly to the previous comment, since `===` produces a boolean, you can just return `letter === string.charAt(0)` directly.